torch/_inductor/__init__.py

```diff
@@ -48,7 +48,9 @@ def aot_compile(
     Returns:
         Path to the generated shared library
     """
-    from .compile_fx import compile_fx_aot
+    from .compile_fx import compile_fx_aot, graph_returns_tuple
+
+    assert graph_returns_tuple(gm), "The FX graph to compile must return a tuple."
```

Comment:
change error to the following:

                "Graph output must be a tuple(). This is so that we can avoid "
                "pytree processing of the outputs. Please change the module to "
                "have tuple outputs or use aot_module instead."

Reply:
I have incorporated your suggestions!